docs/robots/dingo/user_manual_dingo.mdx

```diff
@@ -166,15 +167,10 @@ To connect the shore power, remove the top trough cover (to access the MCU) and
 
 The reference frame used by all Clearpath Robotics ground robots is based on ISO 8855.
 
-<!--- TODO:
```

Comment:
Do you want to change HTML comments with TODO in them? It may be difficult to find these comments in the future. Should these comments be removed?

Reply:
Thoughts?